addons/jobs/jobs-service/src/main/resources/application.properties

```diff
@@ -37,8 +37,13 @@ quarkus.resteasy.gzip.max-input=10M
 quarkus.http.cors=true
 quarkus.http.cors.methods=GET,PUT,POST,DELETE
 quarkus.http.limits.max-body-size=10M
+quarkus.http.port=8080
 
 #Swagger
 quarkus.smallrye-openapi.path=/swagger
 quarkus.swagger-ui.always-include=true
-quarkus.swagger-ui.path=/swagger-ui
\ No newline at end of file
+quarkus.swagger-ui.path=/swagger-ui
+
+#Infinispan - more specific configs on hotrod-client.properties file.
```

Comment:
You can actually use this file for hotrod auth setup which I would prefer as its easier to override

Reply:
yeah, I'll move the config to application.properties.